src/cljc/athens/common_events/resolver/atomic.cljc

```diff
@@ -379,6 +379,36 @@
     tx-data))
 
 
+(defmethod resolve-atomic-op-to-tx :shortcut/new
+  [db {:op/keys [args]}]
+  (let [{page-name :name}    args
+        page-uid             (common-db/get-page-uid db page-name)
+        reindex-shortcut-txs (->> (common-db/get-sidebar-elements db)
+                                  (sort-by :page/sidebar)
+                                  (map-indexed (fn [i m] (assoc m :page/sidebar i)))
+                                  vec)
+        add-shortcut-tx      {:block/uid    page-uid
+                              :page/sidebar (or (count reindex-shortcut-txs)
+                                                1)}
```

Comment:
Shouldn't this be 0? E.g. if there's one existing sidebar element, count would be 1, and the new page would be order 1. So if there's none, it should be order 0, no?

Reply:
Yeah, programmers will say this should be 0 other people will say it should be 1. I don't why this was originally made 1 so I let it be.

Reply:
I don't think this is just an argument of starting number - if it is left at one, the next time a page is added it will also be 1, and the order will thus be incorrect.